controllers/argo.go

```diff
@@ -211,19 +211,19 @@ func applicationName(p api.Pattern) string {
 	return fmt.Sprintf("%s-%s", p.Name, p.Spec.ClusterGroupName)
 }
 
-func getApplication(client argoclient.Interface, name string) (error, *argoapi.Application) {
+func getApplication(client argoclient.Interface, name string) (*argoapi.Application, error) {
```

Comment:
Reason?

Reply:
It's a golang idiom to have the error as the last return variable.